catalogs/catalog-lakehouse-hudi/build.gradle.kts

```diff
@@ -138,3 +150,60 @@ dependencies {
   testRuntimeOnly("org.apache.hudi:hudi-spark$sparkVersion-bundle_$scalaVersion:0.15.0")
```

Comment:
Is it better not to hard code the hudi version file, can we use the version number defined in version.toml?

Reply:
fixed